app/i18n/index.js

```diff
@@ -108,10 +108,26 @@ function loadTranslation(locale) {
     }
 }
 
+let momentLocale = 'en';
+
+function setMomentLocale(locale) {
+    try {
+        if (momentLocale !== locale) {
+            moment.locale(locale);
```

Comment:
I thought `moment.locale()` is deprecated and we should use `updateLocale`, I'm I wrong here?

Reply:
`moment.locale()` is deprecated for updating an existing locale (not for setting which locale is being used currently)

>2.12.0 deprecated using moment.locale() to change an existing locale. Use moment.updateLocale() instead.

Reply:
I tested `moment.locale()` and it's not throwing an error if passed with invalid locale.

On initial run, if invalid locale is passed into it, it returns `en`.  If moment has locale already (say `es`) and passed with invalid locale, fallback is previous locale (which is `es`).

Should this be?
```javascript
momentLocale = moment.locale(locale);
```

Reply:
Yes, that's probably better. I'll update it